app/uk/gov/hmrc/emailverification/models/PasscodeRequest.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.format.ISOPeriodFormat
 import org.joda.time.Period
 import play.api.libs.json.{JsPath, Json, Reads}
 
-case class PasscodeRequest(email:String)
+case class PasscodeRequest(email:String, teamName: String)
```

Comment:
I think `serviceName` would be more accurate than `teamName` (e.g. services move between teams; team names don't necessarily mean anything, etc)